cylc/flow/option_parsers.py

```diff
@@ -83,6 +86,16 @@ def __init__(self, usage, argdoc=None, comms=False, noforce=False,
             else:
                 argdoc = [('REG', 'Suite name')]
 
+        # make comments grey in usage for reasability
```

Comment:
```suggestion
        # make comments grey in usage for readability
```
He heh, misspelling "readability" is almost as hilarious as misspelling "misspelling" :rofl: